### ISSUE TITLE
Old Reports won't Die 

### ISSUE THREAD

**Issue by wriggs** [ushahidi/Ushahidi_iPhone#44]
Old, deleted, reports remain on iPhone after being deleted on web site (http://ushahidi.crowdmap.com)
Refreshing does nothing....

**Comment by stevevg** [comment]
This happens on local installations, too.

**Comment by wriggs** [comment]
Any idea how to fix it?

**Comment by stevevg** [comment]
Nope, I'm not into ObjC programming... I was just bumping an annoying bug ...

**Comment by stevevg** [comment]
In the latest version available, the problem seems bigger. It shows duplicates, does not remove old reports, and does not show new reports done via other means (i.e. website). This app is just  unusable. Maybe before redisigning the UI, devs should address this issue, since till now the app is not usable at all, and seeing so few interest in resolving/answering/asking more details on issues is frankly disappointing, given the possibilites offered by this platform.

**Comment by dalezak** [comment]
Thanks for reporting this issue in the new version. 

This stale report issue has been a problem for awhile and unfortunately the mobile app is at the mercy of the API which in 2.X does not have an elegant way of sharing when reports are removed. 

In order to handle stale reports, the app needs to re-download the entire list of reports and compare it against the current set. I'll investigate a way to calculate this delta but if I can't find an easy solution, will have to resort to dumping the current reports prior to  syncing.

**Comment by stevevg** [comment]
Thank you dale for the prompt answer. I'm a noob in IOS Development, I read the code but I'm not too skilled yet to solve the problem, or I would have willingly helped.

**Comment by r0bm1lls** [comment]
Hi There,
Just to mention that this is still an issue. I'm not sure if anything has changed since the last post. I have set up a deployment in the last week and have similar trouble with stale reports hanging on. Any news?
